# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieParser.h
# Commit: 20708fe16d2a
# Full Hash: 20708fe16d2a1f7e85b609bbfe286ec842b620b4
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-12-18 04:27:40
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 5: Exempt third-party cookie blocking if the channel is on the exception list. r=cookie-reviewers,bvandersloot,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223384
# ==============================================================================

diff -r 7c4b003d1e48 -r 20708fe16d2a netwerk/cookie/CookieParser.h
--- a/netwerk/cookie/CookieParser.h	Tue Dec 17 21:44:37 2024 +0000
+++ b/netwerk/cookie/CookieParser.h	Tue Dec 17 21:44:37 2024 +0000
@@ -50,7 +50,7 @@
              CookieStatus aStatus, nsCString& aCookieHeader,
              const nsACString& aDateHeader, bool aFromHttp,
              bool aIsForeignAndNotAddon, bool aPartitionedOnly,
-             bool aIsInPrivateBrowsing);
+             bool aIsInPrivateBrowsing, bool aOn3pcbException);
 
   bool ContainsCookie() const {
     MOZ_ASSERT_IF(mContainsCookie, mRejection == NoRejection);